### DISCUSSION TITLE
The operation couldn’t be completed. (SkipDriveExternal.ProcessResult.Error error 0.)

### DISCUSSION THREAD

**Discussion by adamvnovak** [skiptools/skip#271] [Q&A] [answered]
<img width="1117" alt="Screenshot 2024-10-25 at 9 39 38 AM" src="https://github.com/user-attachments/assets/baf7b088-b0a5-439b-a9de-4703a77a11c6">

trying to build and run my first skip project.
i ran skip init --open-xcode --appid=com.leaveylabs.cherry-skip cherry-skip CherrySkip
and i’ve started up the android emulator.
but i’m getting this error:

Error: terminated(1): /usr/local/bin/gradle --version output:
The operation couldn’t be completed. (SkipDriveExternal.ProcessResult.Error error 0.)

**Comment by marcprux** [comment]
Can you post the output of `skip checkup`? Also, is this an Intel or ARM Mac?

**Reply by adamvnovak** [reply]
This is an ARM Mac. 

Here is the output of skip checkup: 

Skip thinks Gradle is not installed, but it seems to be installed.
<img width="1409" alt="Screenshot 2024-10-25 at 9 50 56 AM" src="https://github.com/user-attachments/assets/c285478d-1796-4fff-94c3-1cfb406b8f85">

**Reply by adamvnovak** [reply]
Even after reinstalling Gradle Skip says there's an error executing it

**Comment by marcprux** [comment, marked as answer]
I think the issue is that your Homebrew installation was carried over from a previous Intel Mac. Homebrew on ARM installs packages under `/opt/homebrew/bin/`, whereas on Intel it installs under `/usr/local/bin/`. If you run `which gradle`, you will probably see `/opt/homebrew/bin/gradle`. Skip's various tools don't work well under the Rosetta emulation that you are likely falling back to when invoking the various x86-built tools under ARM.

This is a common problem whose easiest solution is to just [uninstall Homebrew](https://docs.brew.sh/FAQ#how-do-i-uninstall-homebrew) and [re-install](https://brew.sh/#install) it from scratch, then install Skip again and ensure that `skip checkup` passes.

**Reply by adamvnovak** [reply]
Thank you! This pointed me in the right direction. It was actually not my Homebrew that was installed in the wrong direction, it was my Java version that was not being referenced properly. I had to reinstall java and things worked properly.
